src/ScottPlot/Plottable/RadarPlot.cs

```diff
@@ -106,6 +106,8 @@ public class RadarPlot : IPlottable
         public int XAxisIndex { get; set; } = 0;
         public int YAxisIndex { get; set; } = 0;
 
+        public float OutlineThickness { get; set; } = 1;
```

Comment:
This is a confusing name, especially with `LineWidth`, but I can't think of a better one. Maybe `DataThickness`?

Maybe Scott will have an idea.

Reply:
I'll split the difference and go with `OutlineWidth` 👍